rabbitmq/manifests/rabbitmq-rc.yaml

```diff
@@ -0,0 +1,66 @@
+#helm:generate helmc tpl -d tpl/values.toml -o manifests/rabbitmq-rc.yaml $HELM_GENERATE_FILE
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: rabbitmq
+  labels:
+    app: rabbitmq
+    provider: rabbitmq-server
+    heritage: bitnami
+spec:
+  replicas: 1
+  selector:
+    app: rabbitmq
```

Comment:
@Fortiz2305 in service images we do not define the `app` label. Only the `provider` label is defined and this case should be `provider: rabbitmq`

f.e. https://github.com/bitnami/charts/blob/master/mariadb/manifests/mariadb-rc.yaml#L16-L19